### ISSUE TITLE
Fails to install on windows due to smaz-py3 error

### ISSUE THREAD

**Issue by ExcaliburZero** [ExcaliburZero/cbpickaxe#2]
Currently if you try to install cbpickaxe on Windows, the pip install fails with the following error:

```
...
  running egg_info
  writing smaz_py3.egg-info\PKG-INFO
  writing dependency_links to smaz_py3.egg-info\dependency_links.txt
  writing top-level names to smaz_py3.egg-info\top_level.txt
  reading manifest file 'smaz_py3.egg-info\SOURCES.txt'
  Traceback (most recent call last):
...
...
...
    File "C:\Users\runneradmin\AppData\Local\Temp\pip-build-env-20yo8oko\overlay\Lib\site-packages\setuptools\command\egg_info.py", line 485, in append
      path = convert_path(item)
             ^^^^^^^^^^^^^^^^^^
    File "C:\Users\runneradmin\AppData\Local\Temp\pip-build-env-20yo8oko\overlay\Lib\site-packages\setuptools\_distutils\util.py", line 139, in convert_path
      raise ValueError("path '%s' cannot be absolute" % pathname)
  ValueError: path '/home/runner/work/smaz-py3/smaz-py3/smazmodule.c' cannot be absolute
  [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error
```

**Comment by ExcaliburZero** [comment]
**Example:** https://github.com/ExcaliburZero/cbpickaxe/actions/runs/6608486205/job/17947254519

**Comment by ExcaliburZero** [comment]
It looks like this issue has been reported before and resolved for smaz-py3, and has maybe re-appeared.

I added a comment to that issue here:

https://github.com/originell/smaz-py3/issues/3#issuecomment-1774392913

**Comment by ExcaliburZero** [comment]
Fixed by the v1.1.3 release of `smaz_py3`.

https://github.com/originell/smaz-py3/issues/3#issuecomment-1776266746

**Comment by ExcaliburZero** [comment]
I added a version bound for smaz-py3 in bbdc1c6e67e3e358db637bd29f8f288ac1fed466
